examples/alignment/tracking/README.md

```diff
@@ -0,0 +1,18 @@
+Tracking for Alignment {#trackingalign}
+======================
+
+1. Edit list of input files
+2. Modify non-input-file variables
+3. Generate JSON enumeration of all hps-mc jobs that need to be run
+```
+hps-mc-job-template -j 1 -a vars.json -i events events.txt 1 job.json.templ jobs.json
+```
+4. Submit generated jobs to batch computing
+```
+hps-mc-batch slurm track_align jobs.json
```

Comment:
I think you should change java_job.py to track_align_job.py to make this submission command work. java_job.py is not the best name, since there are many modes of operation of hps-java and this one in particular is gonna use a RunManager to run a job on an slcio file, so I prefer track_align_job.py personally.

Reply:
Makes sense :) I originally thought it was going to be a more general-purpose job, but it has quickly become more specific to track/align.